app/experimenter/docs/openapi-schema.json

```diff
@@ -2902,6 +2927,56 @@
           }
         }
       },
+      "NimbusExperimentCSV": {
+        "type": "object",
+        "properties": {
+          "launch_month": {
+            "type": "string",
+            "readOnly": true
+          },
+          "product_area": {
+            "type": "string"
+          },
+          "experiment_name": {
+            "type": "string"
+          },
+          "feature_configs": {
+            "type": "string",
+            "readOnly": true
+          },
+          "owner": {
+            "type": "string",
+            "readOnly": true
+          },
+          "start_date": {
+            "type": "string",
+            "readOnly": true
+          },
+          "enrollment_duration": {
+            "type": "string",
+            "readOnly": true
+          },
+          "end_date": {
+            "type": "string",
+            "readOnly": true
+          },
+          "monitoring_dashboard_url": {
+            "type": "string",
+            "readOnly": true
+          },
+          "rollout": {
+            "type": "boolean"
+          },
+          "hypothesis": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "product_area",
+          "experiment_name",
+          "rollout"
+        ]
```

Comment:
I don't think we need `required` anymore. Didn't Jared take this out?

Reply:
they are auto-generated schema, it means that we will always get these values at least `https://swagger.io/specification/`